### ISSUE TITLE
Black square instead of player UI, however player works

### ISSUE THREAD

**Issue by livotov** [bdunn44/MEJS-Vaadin#6]
Hi,

Just installed this component to fresh Vaadin 7.5.1 app with a couple of mp3 files to test. The player itself works and I do hear the music coming from the browser, however the player UI shows as black square, see the picture attached: http://share.livotov.eu/18Gwy/1K9vSWEC.png

But I can control it (pause, resume, skip to next track,etc) without any problems. Any ideas what could be wrong ?

**Comment by Sorbjar** [comment]
Did you set any mediacomponent options?

something like this.
MediaComponentOptions opts = MediaComponentOptions.getDefaultOptions();
        opts.setEnableKeyboard(true);
        opts.setFeatures(new Feature[] { Feature.PROGRESS, Feature.CURRENT,
                Feature.DURATION });
        opts.setAudioWidth(800);
        audioPlayer = new MediaComponent(MediaComponent.Type.AUDIO, opts);

**Comment by livotov** [comment]
nope, I was using vaadin directory page sample without any options. However, adding options did not help, only the player width was adjusted but in other - still the same black rectangle (but still plays :)

Here is my entire init code:

```
Label hello = new Label("Hello Test");
        hello.setStyleName(ValoTheme.LABEL_H1);
        hello.setWidth(null);

        MediaElementPlayerOptions opts = MediaElementPlayerOptions.getDefaultOptions();
        opts.setEnableKeyboard(true);
        opts.setFeatures(new MediaElementPlayerOptions.Feature[]{MediaElementPlayerOptions.Feature.PROGRESS, MediaElementPlayerOptions.Feature.CURRENT, MediaElementPlayerOptions.Feature.DURATION});
        opts.setAudioWidth(800);

        audioPlayer = new MediaElementPlayer(MediaElementPlayer.Type.AUDIO, opts);
        audioPlayer.addPlaybackEndedListener(this);

        btnStart = new Button(" > ");
        btnStop = new Button(" || ");
        btnNext = new Button(" >| ");

        btnStart.setStyleName(ValoTheme.BUTTON_FRIENDLY);
        btnStop.setStyleName(ValoTheme.BUTTON_FRIENDLY);
        btnNext.setStyleName(ValoTheme.BUTTON_FRIENDLY);

        HorizontalLayout hl = new HorizontalLayout();
        hl.setSizeUndefined();
        hl.setMargin(true);
        hl.setSpacing(true);
        hl.addComponent(btnStart);
        hl.addComponent(btnStop);
        hl.addComponent(btnNext);

        VerticalLayout root = new VerticalLayout();
        root.setWidth("100%");
        root.setHeight(null);
        root.setMargin(true);
        root.setSpacing(true);
        root.addComponent(hello);
        root.addComponent(audioPlayer);
        root.addComponent(hl);

        root.setComponentAlignment(hello, Alignment.MIDDLE_CENTER);
        root.setComponentAlignment(audioPlayer, Alignment.MIDDLE_CENTER);
        root.setComponentAlignment(hl, Alignment.MIDDLE_CENTER);

        btnStart.addClickListener(new Button.ClickListener()
        {
            public void buttonClick(final Button.ClickEvent event)
            {
                onStart();
            }
        });

        btnStop.addClickListener(new Button.ClickListener()
        {
            public void buttonClick(final Button.ClickEvent event)
            {
                onStop();
            }
        });

        btnNext.addClickListener(new Button.ClickListener()
        {
            public void buttonClick(final Button.ClickEvent event)
            {
                onNext();
            }
        });

        setContent(root);
```

**Comment by bdunn44** [comment]
This has been fixed in version 1.2.8.
